Source/Tools/Editor/Editor.cpp

```diff
@@ -276,6 +277,20 @@ void Editor::Start()
     context_->GetSubsystem<Engine>()->CreateDebugHud()->SetMode(DEBUGHUD_SHOW_NONE);
 }
 
+int Editor::RunEditorInstance(const ea::vector<ea::string>& arguments, ea::string& output)
+{
+    auto fs = context_->GetSubsystem<FileSystem>();
+
+#if URHO3D_CSHARP && !_WIN32
+    // Editor executable is a C# program interpreted by .net runtime.
+    auto argumentsCopy = arguments;
+    argumentsCopy.push_front(fs->GetProgramFileName());
+    return fs->SystemRun(fs->GetInterpreterFileName(), argumentsCopy, output);
```

Comment:
This isnt the case on linux with .net6 or .net core - they produce native executable like on windows.

Reply:
It is existing code moved from other place, I don't see what I can do about it. Would you fix it?

Reply:
Yeah sure, didnt realize its old code 👍🏻